netmanager/src/redux/MapData/selectors.js

```diff
@@ -11,3 +11,7 @@ export const usePM25SensorData = () => {
 export const useEventsMapData = () => {
   return useSelector((state) => state.mapData.eventsData);
 };
+
+export const useMapSensorsData = () => {
```

Comment:
The same goes for this. How about we have this renamed to `useAllPartnersMapSensorsData` ?